[Content] Remnant: Shattered Light missions (#6761)
endless-sky/endless-sky: data/remnant/remnant 2 side missions.txt

	cargo "equipment" 19
	passengers 13
	source "Viminal"
	destination "Far Monad"
	to offer
		has "Remnant: Shattered Light 2: done"
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Lucina
		ship "Dromedary Specter" "???"
		dialog `It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
	on offer
		conversation
			`The group of people standing around a hologram over on one side of the cafeteria. Ruach gestures for you to join them, and introduces you to each of the people in the group. "I would just like to start by offering our thanks for your assistance, Captain <first>. You have sharp eyes to pick this up as early as you did. Other scouts reported it shortly thereafter, but you spotted it first." She gestures to the hologram of the ship floating in front of you.`
			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the many mysteries here is what has happened to it, and what we should do about it."`

> 			`	Another Remnant steps forward and the hologram changes to show a scene from a cave with a large object. Other Remnant appear to aim some equipment at it, and after a moment or two, the object expands to several times its original size. "These are recordings from our explorations of the ruins, demonstrating the decompression of a Maudrakoi artifact. While not identical, these artifacts generate similar readings to those recorded during your scan of the Shattered Light.`

Reviewer: What's a Maudrakoi? Or explain it's origin (ruins found on Viminal for example).

Author: The name of the aliens whose ruins abound on the Remnant worlds. I'll add some conversation options to let the player ask about it.

> 			`	"As such, we are proposing that we use the same equipment we use for decompression to stabilize the Shattered Light's weft field, and, if possible, bring it fully into synchronity once again." She nods firmly, clapping her hands together to make the hologram vanish.`

Reviewer: You never explain what weft is.

		has "Remnant: Shattered Light 3: done"
	on offer
		fail "Remnant: Shattered Light 3b"
		conversation
			`Eventually you find a good vantage point several kilometers away where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse for the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air lance upwards, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`
			`	The dust and debris settles within a few minutes, and soon you and the <ship> are skimming over the trees. The approach gives you ample time to appreciate just how massive the Shattered Light is. Literally everything that one could need to lay the foundations for a new colony, packed into a single ship. You note large gashes in the hull, along with massive sensor arrays and a shattered dome on top. Based on the mismatch of parts, this ship has been substantially modified from its original hull, and survived untold damage.`

> 			`	As you slow to a hover alongside, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`

Reviewer: Again, primary weft?

				accept
	npc accompany
		government "Remnant (Research)"
		personality launching timid escort
		planet "Far Monad"

> 		ship "Dromedary Wreck" "Shattered Light"

Reviewer 2: When talking with this ship it said that I could not understand what they said (remnant pre understanding their language), is it just because I did the translation missions before the addition of the Remnant(Research) government? Shouldn't there be a patch for that?

Author: Yes, there should be. I'm not sure there is, though. I'll have to look into this.

			choice
				`	"I have things to do, so just let me know if I can help."`
					goto shatteredhelp
				`	"So, what happened to it?"`
			`	"We have had the opportunity to start querying the onboard systems and piece together a little of the Shattered Light's history. As we all know, it was part of the exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`
			`	At this point another Remnant takes up the narrative. "Based on the initial incident logs, it appears that something in the stabilization fields failed during transit. The data is not clear whether the systems failed, or if something in the wormhole fluctuations exceeded the parameters that the systems could handle. In any case, the ship appears to have been shunted laterally from the wormhole in mid-transit.`

> 			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after you spotted it."`

Reviewer: > stuck **out-of-phase** with **prime weft** until an energy surge destabilized it and it **wobbled back** into **prime coherence**

No idea what those words mean in context. How FW introduces hyperlane creating technology (in a way) is how I think you should do this as well. I.E. using simple concepts.

Author: Part of the intent here is to actually create a sort of jarring scenario. These are excited scientists who just did something awesome, and they're forgetting to simplify things as a result.

mission "Remnant: Shattered Light 1"
	name "Shattered Light - Contact"
	invisible
	source
		government "Remnant"
	destination "Viminal"
	to offer
		has "event: remnant: cognizance calm"
		# random < 30
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Caeculus
		ship "Dromedary Ghost" "???"
		dialog "This ship barely registers on your sensors, and you can clearly see through it. The Remnant will want to know about it."
	on complete
		conversation
			`Looking around the Viminal spaceport, you manage to spot Taely using something that looks kind of like a small catapult to throw a large rock across the cavern. A nearby Merganser leaps into the air and chases after it. The ship pounces on the rock, vanishes from sight, and then reappears next to Taely and drops the rock next to her. Without looking away from the Merganser she waves you over.`

> 			`	"Well met, <first>, how goes your travels, found anything interesting?`

Reviewer 2: `how goes your travels` that sounds weird to me but maybe it's correct

Also you're having the Merganser behave like a dog? I love it x)

Author: Thank you! It's a rather fun and visible way to showcase a ship being decidedly not a ship. It might downplay the sentience of the ship a bit; but on the other hand, people play catch or a variety of other games that are basically the same thing, so it fits.

			`	She listens to your description of the encounter with the ghostly ship, and looks at the recording you pull up from your commlink. "This is very intriguing. I will need to consult a few people. If you are interested in investigating this, ensure your ship has a research laboratory installed, and meet me in the starport."`

mission "Remnant: Shattered Light 2"
	name "Searching for Ghosts"
	description "Take Ruach to find and scan the ghostly ship."

> 	blocked "Your commlink chimes with a note that this mission requires a Research Laboratory as well as 2 cargo space and 1 free bunk."

Reviewer 2: I really like this explicit way of saying the mission is blocked!
Never noticed that before in remnant missions, is it new?

Author: I don't recall specifically when I added that; but I think making an effort to have *all* the content the player sees in game actually be treated like in-character text like this adds just a little extra bit of immersion.

	cargo "equipment" 2
	passengers 1
	source "Viminal"
	to offer
		has "Remnant: Shattered Light 1: done"
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Levana
		ship "Dromedary Specter" "???"
		dialog "This ship seems a bit more solid than it did when you first saw it, but it does not seem to be quite present."
	on offer
		require "Research Laboratory"
		conversation

> 			`Your arrival in the starport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They turn as one when you approach, breaking off their conversation.`

Reviewer 3: I want a second opinion.

Reviewer 3: ```suggestion
			`Your arrival in the starport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They both turn when you approach, breaking off their conversation.`
```

> 			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`

Reviewer 3: ```suggestion
			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." `
			
			`Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```

Author: That looks good, except I don't think we do the blank lines in between lines of text.

			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?`
			choice
				`	"OK"`
					accept
				`	"Sorry, not interested."`

> 					decline

Reviewer 2: I guess here you don't get many options because she is in a hurry to see the ship? Maybe mention that, I wish I could ask her about remnant education here ^^
Also did you forget a closing "? (just before choice)

Author: That is correct: She's in a hurry, so not too much conversation here. But I will keep that in mind as a topic to include. I do intend to do follow-up missions to the Shattered Light set, so I'll remember to include a bit on education at that time.

And yes, I did miss a closing quotation mark there. Thanks!

	on complete
		payment 50000
		conversation

> 			`The <ship> has not even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple spots down to stare up at the gills of an Albatross. A few moments later, Taely drops down to the ground beside her. As you approach you can see Ruach has a hologram of the spectral ship floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach says as you approach.`

Reviewer 3: ```suggestion
			`The <ship> hasn't even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple of spots down to stare up at the gills of an Albatross. Taely copied her actions a few moments later, taking her time as you make preparations to follow them. You can see Ruach has a hologram of the spectral ship by the time you arrive, floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach mused.`
```

> 			`	"Sorry for leaving you behind, this is just such an exciting discovery. This ship you have found matches the profile of the Shattered Light - a large city ship that was part of the exodus but never arrived on this side of the wormhole. There are so many questions here! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`

Reviewer 3: ```suggestion
			`	"Sorry for leaving you behind, but this is just such an exciting discovery. This ship you found matches the profile of the Shattered Light - a large city ship that was part of the exodus but never arrived on this side of the wormhole. There are so many questions to be answered! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
```

Reviewer 3: I am tempted to add an action before "Anyway," but I'm running a bit tight on time right now, so go figure.

			

mission "Remnant: Shattered Light 3"
	invisible
	name "Ghost Touch"
	description "Help the Remnant experiment on the Shattered Light."
	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 cargo."
	cargo "equipment" 19
	passengers 13
	source "Viminal"
	destination "Far Monad"
	to offer
		has "Remnant: Shattered Light 2: done"
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Lucina
		ship "Dromedary Specter" "???"
		dialog `It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
	on offer
		conversation

> 			`The group of people standing around a hologram over on one side of the cafeteria. Ruach gestures for you to join them, and introduces you to each of the people in the group. "I would just like to start by offering our thanks for your assistance, Captain <first>. You have sharp eyes to pick this up as early as you did. Other scouts reported it shortly thereafter, but you spotted it first." She gestures to the hologram of the ship floating in front of you.`

Reviewer 3: ```suggestion
			`A group of people huddled around a hologram catches your attention, drawing your eyes to Rauch who gestures for you to join them. She introduces you to each person, before thanking you: "I would just like to start- blah blah blah I don't care about the dialogue part, blah- pick this up as early as you did." She gestures to the hologram of the ship floating in front of you. "Other scouts report- you spotted it first."`
```

> 			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the many mysteries here is what has happened to it, and what we should do about it."`

Reviewer 3: ```suggestion
			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the oldest mysteries here is what has happened to it, and what we should do about it."`
```

			`	Another Remnant steps forward and the hologram changes to show a scene from a cave with a large object. Other Remnant appear to aim some equipment at it, and after a moment or two, the object expands to several times its original size. "These are recordings from our explorations of the ruins, demonstrating the decompression of a Maudrakoi artifact. While not identical, these artifacts generate similar readings to those recorded during your scan of the Shattered Light.`

> 			`	"As such, we are proposing that we use the same equipment we use for decompression to stabilize the Shattered Light's weft field, and, if possible, bring it fully into synchronity once again." She nods firmly, clapping her hands together to make the hologram vanish.`

Reviewer 3: ```suggestion
			`	"As such, our proposition is to use the same equipment we used for decompression to stabilize the Shattered Light's weft field, and, if possible, bring it fully into synchronicity once again." She nods firmly, clapping her hands together to make the hologram vanish.`
```

			`	Her gestures take on an appreciative tone. "The Maudrakoi made extensive use of the materials that make our ships and architecture so distinctive, and this has allowed their cities and technologies to survive the eons since they went extinct. One technology of note is they had an interesting ability to manipulate space. They used this to make large, highly complex objects, and then, for lack of a better term, partially embed them in pocket dimensions so as to not be entirely present. They used this frequently to allow equipment and materials to be much smaller than normal." She shrugs, and gestures with a frustrated tone. "We cannot replicate it, but we have figured out how to disrupt the compression, and thus recover the artifact."`
			`	The hologram shifts back to showing the Shattered Light. "This ship does not show the same signatures as the Maudrakoi artifacts, but we think it is sufficiently similar that our equipment should work on it too. Hopefully. In any case, though, time is passing, so we should start loading." With that, the Remnant gather their equipment and head out.`
				accept
	on complete
		conversation

> 			`With the Shattered Light careening towards the surface of Far Monad, you push the <ship> into a dive. As you focus on finding a safe place to land with a view of the approaching ship, you notice out the corner of your eye that the Remnant team have all suited up and are pushing their equipment to the cargo hatch. A message appears on your console.`

Reviewer 3: ```suggestion
			`With the Shattered Light careening towards the surface of Far Monad, you push the <ship> into a dive. As you focus on finding a safe place to land within close proximity of the approaching ship, you notice out the corner of your eye that the Remnant team have all suited up and are pushing their equipment to the cargo hatch. A message appears on your console.`
```

			`	"Ready for drop, Captain. Just get us along side the Shattered Light without too much forward velocity, we can handle the rest."`
			`	Moments later your ship pings a point on the ship indicating a gaping hole on the ship. Nodding your assent, the <ship> swoops to a hold position just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment falling into it on repulsor pallets while the Remnant drop alongside it. A final message appears on your console.`
			`	"Drop success. Get clear of the arrival zone and pick us up after the dust settles."`
			`	The <ship> veers and accelerates away from the approaching wreck without waiting for your approval and retreats to a safe distance.`

mission "Remnant: Shattered Light 3b"
	name "Remnant: Ghost Touch"
	description "Help the Remnant experiment on the Shattered Light."
	source "Viminal"
	destination "Viminal"
	waypoint "Coluber"
	to offer
		has "Remnant: Shattered Light 2: done"
	to complete

> 		has "Remnant: Shattered Light 3: done"

Author: Shattered Light 3 is an invisible mission that actually goes to the planet. The people giving the mission are expecting this mission to be simply a "Go out, find the ship somewhere near Coluber, try out the equipment, and come back."

As such, this mission is the visible one with the false waypoints that tells the player where the Remnant think the mission is going; while the main mission is the one that turns out to be a regular point to point mission going to Far Monad.

> mission "Remnant: Shattered Light 4"

Reviewer 3: I ran out of time. This is the last part I reviewed at. 

_Don't you dare remove this till I start a new review chain._

	name "Shattered Light"
	description "Escort the wreck back to Viminal."
	landing
	source "Far Monad"
	destination "Viminal"
	to offer
		has "Remnant: Shattered Light 3: done"
	on offer
		fail "Remnant: Shattered Light 3b"
		conversation

> 			`Eventually you find a good vantage point several kilometers away where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse for the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air explode out from it, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`

Author: You are basically watching a ghost fall through the sky, with the disconcerting effect that clouds and whatnot go *through* the ship.

> 			`	The dust and debris settles within a few minutes, and soon you and the <ship> are skimming over the vegetation. The approach gives you ample time to appreciate just how massive the Shattered Light is. Literally everything that one could need to lay the foundations for a new colony, packed into a single ship. You note large gashes in the hull, along with massive sensor arrays and a shattered dome on top. Based on the mismatch of parts, this ship has been substantially modified from its original hull, and survived untold damage.`

Reviewer 3: ```suggestion
			`	The dust and debris settle within a few minutes, and soon you and the <ship> are skimming over the vegetation. The approach gives you ample time to appreciate just how massive the Shattered Light is; Everything that one could need to lay the foundations for a new colony is packed into a single ship. You note large gashes in the hull, massive sensor arrays and a shattered dome on top. Based on the mismatch of parts, this ship has been substantially modified from its original hull, and survived untold damage.`
```

			`	As you slow to a hover alongside, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`
				accept
	npc accompany
		government "Remnant (Research)"
		personality launching timid escort
		planet "Far Monad"
		ship "Dromedary Wreck" "Shattered Light"
	npc evade
		government "Korath"
		personality entering plunders harvests
		system Lucina
		fleet "Korath Ember Waste Raid" 1
	on complete
		payment 250000
		clear "Remnant: Shattered Light 3b: failed"
		set "Remnant: Shattered Light 3b: done"
		conversation

> 			`	The Shattered Light is so big that traffic control directs the two of you to a large valley next to the spaceport, where you discover that a section of chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`

Reviewer 3: ```suggestion
			`	The Shattered Light is so massive that traffic control directs your ships to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
```

Author: your ships doesn't quite sound right. maybe just `directs you to a large valley`? After all, the SL is not one of your ships, and there's no requirement the player have more than one.

			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo hold, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them: They are not self-aware, sentient, or even particularly advanced artificial intelligence. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`
			scene "outfit/android"
			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`
		outfit "NDR-114 Android" 5

mission "Remnant: Shattered Light 5"
	name "Brilliant Energy"
	description "Retrieve equipment from <destination> to help research the Shattered Light."
	landing
	source "Viminal"
	destination "Caelian"

> mission "Remnant: Shattered Light 6"

Author: Uh... Something seems to have eaten it. That's a problem. Not a huge one, but even so... I'll look into this.

> mission "Remnant: Shattered Light 6"

Author: Alright, dug into this and re-discovered what's going on. Namely, that all the conversation that starts this mission is part of the conclusion conversation for Shattered Light 4. Conversely, all the complete text is part of the on offer text for SL6.

This mission literally only exists so that there's a map marker and something in the mission list so that player's don't forget where they are supposed to be going.

			`	The entire complement of Remnant crowd the windows when you finally settled into final approach for the covered valley that houses the Shattered Light. You can see that the Remnant have been busy setting up large equipment towers around it, including several small encampments of tents. As you settle in for a landing, you see a camel team maneuver a sled with a large anti-aircraft battery into position, where it is promptly shrouded in a tent that blends into the rest of the camp.`
			`	Traffic control directs you to a pad near a large door of the Shattered Light. Ruach and a small group of Remnant are gathered around a hologram of the ship, and seem to be co-ordinating teams exploring it. Large mounds of equipment appear to have expanded from the sides of the ship, some of which appears to be functional. As you go through your landing checks, the team thank you for the transportation and begins offloading their equipment.`
			`	"Welcome back, <first>!" signs Ruach from across the pad. "We have made considerable progress already." She appears set to explain what they have discovered.`
			choice

> 				`	"Great, but please save your explanations for those who want them."`

Reviewer 2: ```suggestion
				`	"Great, but please spare me the details."`
```
Maybe a bit too rough? Could be something conveying the feeling "I don't follow science things" too

					goto shatteredrequest

> 				`	"You have figured out what happened? Great!"`

Reviewer 3: ```suggestion
				`	"You figured out what happened? Great!"`
```

			`	She looks crestfallen "Well, we know what the ship thinks happened, anyway. It is likely that what actually happened differs significantly." Her posture takes on a more positive stance. "We have had the opportunity to start querying the onboard systems and piece together a little of the Shattered Light's history. Particularly the logs.`
			`	"As we all know, it was part of the exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`
			`	At this point another Remnant takes up the narrative. "Based on the initial incident logs, it appears that something in the stabilization fields failed during transit. The data is not clear whether the systems failed, or if something in the wormhole fluctuations exceeded the parameters that the systems could handle. In any case, the ship appears to have been shunted laterally from the wormhole in mid-transit.`

> 			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after you spotted it."`

Reviewer 3: ```suggestion
			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after, you spotted it."`
```

			choice
				`	"Interesting. Well, let me know if I can help."`
					goto shatteredhelp
				`	"I am curious as to what you will do now."`

> 			`	"The first step will be to unpack the Shattered Light and setup all its equipment. Taking detailed scans of each piece as we do so, of course." She gestures around at the equipment expanding out of the side of the ship. "As you can see, we have already started the process, and found that much of it is operational. One of the other Remnant continues. "The Shattered Light carried what amounts to an entire heavy industrial factory. Everything from smelters to welding equipment and everything in between." They smile and their gestures take on a tone of irony. "In fact, the ships we have now are in no small part due to the Shattered Light's absence. Without these capabilities, the the choice about whether or not to commit to learning the techniques of those who lived here before us was much easier. If the Shattered Light had arrived on schedule we may well still be building ships out of the traditional materials prevalent at the time."`

Reviewer 3: ```suggestion
			`	"The first step will be to unpack the Shattered Light and set up all its equipment. Taking detailed scans of each piece as we do so, of course." She gestures around at the equipment expanding out of the side of the ship. "As you can see, we have already started the process and found that much of it is operational. One of the other Remnant continues. "The Shattered Light carried what amounts to an entire heavy industrial factory. Everything from smelters to welding equipment and everything in between." They smile and their gestures take on a tone of irony. "In fact, the ships we have now are in no small part due to the Shattered Light's absence. Without these capabilities, the choice about whether or not to commit to learning the techniques of those who lived here before us was much easier. If the Shattered Light had arrived on schedule we may well still be building ships out of the traditional materials prevalent at the time."`
```

Reviewer 3: "the the"

				goto shatteredrequest
			label shatteredhelp
			`	"Thank you, captain. Of course we do not want to keep you waiting, so rest assured that we will contact you if your assistance is needed in the future."`
			label shatteredrequest

> 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work; and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`

Reviewer 3: ```suggestion
			`	Taely gestures take a tone of curiosity. "One thing that you could help us with if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
```

			label mainreturn
			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles micheviously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`

mission "Remnant: Shattered Light 1"
	name "Shattered Light - Contact"
	invisible
	source
		government "Remnant"
	destination "Viminal"
	to offer
		has "event: remnant: cognizance calm"

> 		# random < 30

Author: It's not essential, no; but I do like having the exact timing of when it starts floating around in the Ember Waste to be somewhat uncertain.

		personality uninterested waiting staying mute
		system Caeculus
		ship "Dromedary Ghost" "???"
		dialog "This ship barely registers on your sensors, and you can clearly see through it. The Remnant will want to know about it."
	on complete
		conversation
			`Looking around the Viminal spaceport, you spot Taely using something that slightly resembles a small catapult to launch a sizable rock across the cavern. A nearby Merganser leaps into the air and chases after it, pouncing on the rock and vanishing from sight before reappearing next to Taely, dropping it next to her. Without looking away from the Merganser she waves you over.`
			`	"Well met, <first>, how goes your travels, found anything interesting?`
			`	She listens to your description of the encounter with the ghostly ship whilst looking at the recording you pull up from your commlink. "This is very intriguing. I will need to consult a few people. If you are interested in investigating this, ensure your ship has a research laboratory installed, and meet me in the starport."`

mission "Remnant: Shattered Light 2"
	name "Searching for Ghosts"
	description "Take Ruach to find and scan the ghostly ship."

> 	blocked "Your commlink chimes with a note that this mission requires a Research Laboratory as well as 2 cargo space and 1 free bunk."

Author: I'm not sure what to say. It should work, and it isn't generating any errors, but I will compare it to some other examples to make sure I got the formatting right.

	on offer
		require "Research Laboratory"
		conversation
			`Your arrival in the starport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They both turn when you approach, breaking off their conversation.`
			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." `
			`Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?"`
			choice
				`	"OK"`
					accept
				`	"Sorry, not interested."`
					decline
	on complete
		payment 50000
		conversation

> 			`The <ship> hasn't even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple of spots down to stare up at the gills of an Albatross. Taely copied her actions a few moments later, taking her time as you make preparations to follow them. You can see Ruach has a hologram of the spectral ship by the time you arrive, floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach mused.`

Author: I think this got erroneously changed. Taely is up in the gills performing maintenance; and this is supposed to be a subtle sort of "How did Ruach know that's where Taely was?" sort of thing. I'll have to change this.

> 			`The <ship> hasn't even settled onto the landing pad when Ruach leaps down from the airlock and jogs a couple of spots down to stare up at the gills of an Albatross. Taely copied her actions a few moments later, taking her time as you make preparations to follow them. You can see Ruach has a hologram of the spectral ship by the time you arrive, floating in the air between them. "It is lacking the confirmation codes or identification info, but I am confident that this is it." Ruach mused.`

Author: Alright, fixed.

			`	Another Remnant steps forward and the hologram changes to show a scene from a cave with a large object. Other Remnant appear to aim some equipment at it, and after a moment or two, the object expands to several times its original size. "These are recordings from our explorations of the ruins, demonstrating the decompression of a Maudrakoi artifact. While not identical, these artifacts generate similar readings to those recorded during your scan of the Shattered Light.`
			`	"As such, our proposition is to use the same equipment we used for decompression to stabilize the Shattered Light's weft field, and, if possible, bring it fully into synchronicity once again." She nods firmly, clapping her hands together to make the hologram vanish.`
			`	Ruach takes the lead again. "So, Captain, we will need you to bring us in close to the Shattered Light so we can give this a try. Our sentries have reported seeing it in the systems near Coluber, so it may take some searching."`
			choice
				`	"Looks good. Start loading the <ship>!"`
					accept
				`	"Wait, 'Maudrakoi'?"`

> 			`	The Remnant who showed the video looks startled, and then embarrassed. "My apologies, I did not realize you had not been familiarized with them." They look around the room, then gesture and their hologram shifts to show a large room with many shelves around the edge and several large objects within it, including some that look like point defense turrets.`

Reviewer 4: Any particular reason gender neutral prnouns are used here when feminine ones are used both before and affter this line? Also, this hologram was just turned off.
```suggestion
			`	The Remnant who showed the video looks startled, and then embarrassed. "My apologies, I did not realize you had not been familiarized with them." She look around the room, then gesture and the hologram returns, now showing a large room with many shelves around the edge and several large objects within it, including some that look like point defense turrets.`
```

Author: Not a strong reason, no. I tend to drift towards the neutral when the speaker's gender isn't an important detail, or when I prefer to shift the focus on the description.

			`	"Ready for drop, Captain. Just get us along side the Shattered Light without too much forward velocity, we can handle the rest."`
			`	Moments later your ship pings a point on the ship indicating a gaping hole on the ship. Nodding your assent, the <ship> swoops to a hold position just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment falling into it on repulsor pallets while the Remnant drop alongside it. A final message appears on your console.`
			`	"Drop success. Get clear of the arrival zone and pick us up after the dust settles."`
			`	The <ship> veers and accelerates away from the approaching wreck without waiting for your approval and retreats to a safe distance.`

mission "Remnant: Shattered Light 3b"
	name "Remnant: Ghost Touch"

> 	description "Help the Remnant experiment on the Shattered Light."

Reviewer 4: ```suggestion
	description "Help the Remnant experiment on the Shattered Light. It was last seen in the Coluber system. Track it down and carry out an outfit scan."
```

	source "Viminal"
	destination "Viminal"
	waypoint "Coluber"
	to offer
		has "Remnant: Shattered Light 2: done"
	to complete
		has "Remnant: Shattered Light 3: done"

mission "Remnant: Shattered Light 4"
	name "Shattered Light"
	description "Escort the wreck back to Viminal."
	landing
	source "Far Monad"
	destination "Viminal"
	to offer
		has "Remnant: Shattered Light 3: done"
	on offer
		fail "Remnant: Shattered Light 3b"
		conversation

> 			`Eventually you find a good vantage point several kilometers away where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse by the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air explode out from it, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`

Reviewer 4: ```suggestion
			`Eventually, you find a good vantage point, several kilometers away, where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse by the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air explode out from it, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`
```

			scene "outfit/android"
			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`
		outfit "NDR-114 Android" 5

mission "Remnant: Shattered Light 5"
	name "Brilliant Energy"
	description "Retrieve a team with their equipment from <destination> to help research the Shattered Light."
	landing
	source "Viminal"
	destination "Caelian"
	to offer
		has "Remnant: Shattered Light 4: done"

mission "Remnant: Shattered Light 6"
	name "Shattered Light"
	description "Ferry the team and their gear back to <destination>"
	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 cargo."
	cargo "equipment" 18
	passengers 11
	landing
	source "Caelian"
	destination "Viminal"
	to offer

> 		has "Remnant: Shattered Light 5: done"

Reviewer 4: ```suggestion
		has "Remnant: Shattered Light 5: offered"
```
Or:
```suggestion
		has "Remnant: Shattered Light 4: done"
```

Author: No, SL 5 should complete as normal, and be used as a "done" mission to test for here. In the absence of any "to complete" conditions, it will complete merely with the act of arriving at Caelian; after which SL 6 can trigger.

Reviewer 4: It looks like the problem I was having was because of the lack of `accept` on the offer conversation.

	on offer
		conversation
			`On <destination> the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`
			`	The team you are sent to pick up are waiting next to the information desk. They have a collection of pallets nearby, and a small crowd of people is gathered around them in animated conversation. They spot you and the crowd start waving their goodbyes as the group pick up their bags and head towards you.`

> 			`	"Ah, <first>! Thank you for transport us. We are excited to see the Shattered Light and start studying this mystery you have found." They briskly head to your ship.`

Reviewer 4: ```suggestion
			`	"Ah, <first>! Thank you for transport us. We are excited to see the Shattered Light and start studying this mystery you have found." They briskly head to your ship.`
				accept
```

			`	"Ready for drop, Captain. Just get us along side the Shattered Light without too much forward velocity, we can handle the rest."`
			branch livingship
				or
					has "flagship model: Peregrine"
					has "flagship model: Merganser"
			`	Moments later you spot a gaping hole on the ship. Focusing on it, you fly the <ship> down and hold position just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment falling into it on repulsor pallets while the Remnant drop alongside it. A final message appears on your console.`
				goto dropsuccess
			label livingship
			`	Moments later the <ship> pings a point on the Shattered Light indicating a gaping hole on the ship. Nodding your assent, your ship swoops to a hold position just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment falling into the hole on repulsor pallets while the Remnant drop alongside it. A final message appears on your console.`
			label dropsuccess
			`	"Drop success. Get clear of the arrival zone and pick us up after the dust settles."`

> 			`	The <ship> veers and accelerates away from the approaching wreck without waiting for your approval and retreats to a safe distance.`

Reviewer 4: This line should also be different based on flagship.

Author: Done!

			label mainreturn
			`	Taely leads you around the area for a few more hours, pointing out historical features, places where they resurrected trades and skills from before the advent of flight, and other interesting things. Eventually she heads back to the train station, which deposits you back in the main spaceport.`
			`	"I hope you enjoyed the tour, <first>. It was interesting to see our history through fresh eyes." She smiles micheviously. "And I enjoyed the hoverboarding, too. Maybe we can do it again sometime." She pauses for a moment, focusing on something you cannot see. "Looks like duty calls. Embers guide your way, Captain <last>." She waves, and heads off into the crowd.`

mission "Remnant: Shattered Light 1"
	name "Shattered Light - Contact"
	invisible
	landing
	source
		government "Remnant"
	destination "Viminal"
	to offer
		has "event: remnant: cognizance calm"

> 		# random < 30

Author: The comment is only there for ease of people testing. Will remove it now.

> 	npc "scan outfits"

Author: I'd like to think that people that are this far into the Remnant storyline *may* have a bit of curiosity in them and wonder what this ghostly ship they've *never* seen before is. And if they don't... Well, this is a side mission for a reason.

> 	npc "scan outfits"

Reviewer 5: Personally, I scan any ship that looks interesting to me, and a ghost ship certainly qualifies as interesting.

	cargo "equipment" 2
	passengers 1
	source "Viminal"
	to offer
		has "Remnant: Shattered Light 1: done"
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Levana
		ship "Modified Dromedary Specter" "???"
		dialog "This ship seems a bit more solid than it did when you first saw it, but it does not seem to be quite present."
	on offer
		require "Research Laboratory"
		conversation
			`Your arrival in the spaceport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They both turn when you approach, breaking off their conversation.`
			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." `
			`Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`

> 			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?"`

Author: Thanks for the suggestion. I have an "on enter" for this. It'll be included in my next push.

			

mission "Remnant: Shattered Light 3"
	invisible
	name "Ghost Touch"
	description "Help the Remnant experiment on the Shattered Light."
	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 tons of cargo."
	cargo "equipment" 19
	passengers 13
	source "Viminal"
	destination "Far Monad"
	to offer
		has "Remnant: Shattered Light 2: done"
	npc "scan outfits"
		government "Ember Waste"
		personality uninterested waiting staying mute
		system Lucina
		ship "Modified Dromedary Specter" "???"
		dialog `It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`
	on offer
		conversation
			`A group of people huddled around a hologram catches your attention, drawing your eyes to Ruach who gestures for you to join them. She introduces you to each person, before thanking you. "I would just like to start by offering our thanks for your assistance, Captain <first>. You have sharp eyes to pick this up as early as you did." She gestures to the hologram of the ship floating in front of you. "Other scouts reported it shortly thereafter, but you spotted it first."`

> 			`	"We have confirmed that this matches all available records of the Shattered Light, although it appears to have suffered considerable damage. One of the oldest mysteries here is what has happened to it, and what we should do about it."`

Author: The same person, so the closing quotation mark will be removed.

			`	"We do not have time for an in-depth presentation, but the short version is this: the Maudrakoi are the primary civilization that lived on these worlds before we arrived. They are not the only civilization to have lived here, but they are the one we find most often in the archeological record. Fortunately for our research, it seems that they lived extensively underground in complexes such as this one."`
			`	Her gestures take on an appreciative tone. "The Maudrakoi made extensive use of the materials that make our ships and architecture so distinctive, and this has allowed their cities and technologies to survive the eons since they went extinct. One technology of note is they had an interesting ability to manipulate space. They used this to make large, highly complex objects, and then, for lack of a better term, partially embed them in pocket dimensions so as to not be entirely present. They used this frequently to allow equipment and materials to be much smaller than normal." She shrugs, and gestures with a frustrated tone. "We cannot replicate it, but we have figured out how to disrupt the compression, and thus recover the artifact."`

> 			`	The hologram shifts back to showing the Shattered Light. "This ship does not show the same signatures as the Maudrakoi artifacts, but we think it is sufficiently similar that our equipment should work on it too. Hopefully. In any case, though, time is passing, so we should start loading." With that, the Remnant gather their equipment and head out.`

Reviewer 6: This seems like a very heavy and complex lore dump to me. So we've known that there were civilizations in this area of the Ember Waste prior to the Remnant. But now we're getting a name--Is this just what the Remnant call them or is this name something they discovered? Is it someone we know of, but just by a different name?--and a description of highly advanced tech that we haven't seen from anywhere else. Not only that, but the Remnant have somehow been able to understand enough about this tech to reverse its effects, all at the same time while dropping a reference to the weft, another idea that hasn't been mentioned anywhere else in the game.

I'm willing to let this slide for now, but I can very easily see us having to come back to this in the future to tighten things up.

			`	As you slow to a hover alongside, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`
				accept
	npc accompany
		government "Remnant (Research)"
		personality launching timid escort
		planet "Far Monad"
		ship "Dromedary Wreck" "Shattered Light"
	npc evade
		government "Korath"
		personality entering plunders harvests
		system Lucina
		fleet "Korath Ember Waste Raid" 1
	on complete
		payment 250000
		clear "Remnant: Shattered Light 3b: failed"
		set "Remnant: Shattered Light 3b: done"
		conversation

> 			`	The Shattered Light is so massive that traffic control directs you to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now, however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`

Reviewer 7: ```suggestion
			`	The Shattered Light is so massive that traffic control directs you to a large valley next to the spaceport, where you discover that a section of the chasm has been covered to create a semi-enclosed area. Judging from the equipment and stacks of freight, this area looks like it is predominantly used by bulk freight shipments and little else. Now however, as the Shattered Light slowly moves in, a flock of Pelicans scatter in front of it looking like nothing more than a flock of plovers in front of an approaching camel.`
```

Author: I'm not sure about this one. "Now however," doesn't feel right to me. I do intend this to be "Now, pause, however, pause, as the shattered light...." It's not "Now however, pause, ....

Reviewer 7: Yeah I'm not super attached to this suggestion, you can resolve it if you like.

			`	Down on the ground, you see Ruach and her team exit an airlock on the side of the Shattered Light and jump down to the ground. They gather around, giving hugs. "<first>!" exclaims Ruach. "We have recovered a lost piece of Remnant history. Thank you for helping. I do not know what this will mean for us in the long term, but there will be a lot of careful examination for the next few weeks."`
			choice
				`	"I have things to do, so just let me know if I can help."`
					goto shatteredhelp
				`	"So, what happened?"`

> 			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their gestures are comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`

Reviewer 7: ```suggestion
			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their signing is comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`
```

			`	He frowns a bit. "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the primary weft, so now we have time to work on it and figure out what happened to it.`
			choice
				`	"I think I get the picture. Let me know if I can help."`
					goto shatteredhelp

> 				`	"What is this 'Primary Weft' you have mentioned?"`

Reviewer 6: "Weft" and "Primary Weft" aren't capitalized up until you ask this question, but then it sometimes goes back to lower case after this.. Should they always be capitalized or not?

Author: Alright, so I went back through this, and there's basically three utilization's here:

Primary Weft. This is a proper noun, a named specific thing, what the Remnant call the dimension that we live in.
Secondary Weft. This is also a proper noun, a specific thing, what the Remnant call the dimension that the Shattered Light is partially in.

weft. This is a general noun. Not actually synonymous with "dimension" but reasonably similar. So if they're talking about the weft, then they're talking about it in a general sense; For instance, they consider generating a weft field to manipulate how it interacts with the Primary and Secondary Wefts, sort of like a warp field, but obviously different functionalities and effects.

It is possible that there are instances of The Weft that I have missed, which would shortened forms for whatever weft the speaker is focusing on, but probably the Primary Weft. 

(so weft vs Weft is kind of a "There's lots of trees, but that one is The Tree that I care about." sort of thing.)

> 			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort."`

Reviewer 8: Alright, I don't have time to get into this in detail right now, but this is a lot of heavy cosmological lore to drop into a casual side mission. It is also a departure from the lore document despite pulling in a piece of terminology for it, used incorrectly.

This is not necessarily a problem, the lore document was only intended as MZ's notes and we're not _obliged_ to conform to it. It does; however, have some significant implications for the discussion around what hyperlanes are and how they work - with some departures from common assumptions.

On all accounts it at least warrants close scrutiny.

Author: Definitely. It is meant to provoke thought, after all. That being said, it is all presented as a mix of what they think it's true and what they hypothesize is true, so I don't think it should be a problem either way. :) They might be the most knowledgeable humans on the subject, but they are nowhere near the understanding of the Quarg or Pug our Korath.

Author: I've tweaked this reference a bit to add in a mention of the "warp" and introduce a bit of discussion into the matter. So now it isn't just the Remnant stating "this is the name," but rather them saying "This is what we've called it, but we're thinking this other name might fit better." In other words, it is a growing field.

> 			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the secondary weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the primary weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort. Actually, it has been proposed that we call the Primary Weft the Warp. It is an old weaving term that serves as a reasonable analogy to how the warp and weft interact, and might better reflect its status instead of being labeled 'Primary.'`

Reviewer 6: So we're introducing the terms Warp and Weft that were explained in the lore dump as being ideas from the Drak, but these don't seem to be the same definitions that were used there. To the Drak, the Warp and Weft were two different dimensions/planes of existence. But here, we have a Remnant saying that the "Primary Weft" and "Warp" are the same thing.

Perhaps the Remnant have these ideas of existence, but if they do I don't think they should be using the same terms that we know the Drak to have used (or at least whatever they would have said in the Drak language as it translates to English).

Author: The Remnant have different ideas about how things work, and I kind of like the idea of the lore doc version being more right than the Remnant one, but I deliberately want the cross-contamination of words & terms here.

			label shatteredhelp
			`	"In terms of help, could you go to Caelian? We have sent word ahead, and our engineers are retrieving some of our remaining equipment from the time of the Exodus. We are hoping that it might be of use to help us repair the Shattered Light, or at least understand what happened to it.`

> 			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them: They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`

Reviewer 7: ```suggestion
			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them. They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, so we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`
```

> 			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment. "These are known as NDR-114 Androids, and were cutting edge innovations back when the exodus occurred. Don't expect too much of them. They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, so we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship."`
> 			scene "outfit/android"
> 			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`

Reviewer 6: Just moving the android scene up and capitalizing Exodus.
```suggestion
			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment.`
			scene "outfit/android"
			`	 "These are known as NDR-114 Androids, and were cutting edge innovations back when the Exodus occurred. Don't expect too much of them. They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, so we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship.`
			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`
```

		outfit "NDR-114 Android" 5

mission "Remnant: Shattered Light 5"
	name "Brilliant Energy"
	description "Retrieve a team with their equipment from <destination> to help research the Shattered Light."
	landing
	source "Viminal"
	destination "Caelian"
	to offer
		has "Remnant: Shattered Light 4: done"

mission "Remnant: Shattered Light 6"
	name "Shattered Light"
	description "Ferry the team and their gear back to <destination>"

> 	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 cargo."

Reviewer 7: ```suggestion
	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 tons of cargo."
```

	cargo "equipment" 18
	passengers 11
	landing
	source "Caelian"
	destination "Viminal"
	to offer
		has "Remnant: Shattered Light 5: done"
	on offer
		conversation
			`On <destination> the spaceport is awash in conversation, and you spot snippets here and there of people talking about the recovery of the Shattered Light. Many are speculating as to what happened to it in the first place, while others are discussing the technical merits of what the research team did to recover it. Details travel quickly, it seems.`

> 			`	The team you are sent to pick up are waiting next to the information desk. They have a collection of pallets nearby, and a small crowd of people is gathered around them in animated conversation. They spot you and the crowd start waving their goodbyes as the group pick up their bags and head towards you.`

Reviewer 6: ```suggestion
			`	The team you are sent to pick up are waiting next to the information desk. They have a collection of pallets nearby, and a small crowd is gathered around them in animated conversation. They spot you and the crowd start waving their goodbyes as the group pick up their bags and head towards you.`
```

> 			`	"Ah, <first>! Thank you for transport us. We are excited to see the Shattered Light and start studying this mystery you have found." They briskly head to your ship.`

Reviewer 7: ```suggestion
			`	"Ah, <first>! Thank you for transporting us. We are excited to see the Shattered Light and start studying this mystery you have found." They briskly head to your ship.`
```

				accept
	on complete
		event "remnant: shattered light"
		event "remnant: shattered light research" 60
		conversation
			`The Remnant spend the trip busily discussing what is known so far and exchanging theories, eagerly asking you questions about what you saw and reviewing your sensor records. By the time you reach Viminal, they have a whole collection of theories and have a preliminary plan of what studies they hope to do. When pressed, however, they all acknowledge that their top priority will be documenting the ship. Every scratch, every oddly charged component, every minor modification. "After all," they say, "knowing how it is now is the first step to determining how it arrived here."`

> 			`	The entire complement of Remnant crowd the windows when you finally settle into the final approach for the covered valley that houses the Shattered Light. You can see that the Remnant have been busy setting up large equipment towers around it, including several small encampments of tents. As you settle in for a landing, you see a camel team maneuver a sled with a large anti-aircraft battery into position, where it is promptly shrouded in a tent that blends into the rest of the camp.`

Reviewer 6: ```suggestion
			`	The entire complement of Remnant crowd the windows when you settle into the final approach for the covered valley that houses the Shattered Light. You can see that the Remnant have been busy setting up large equipment towers around it, including several small encampments of tents. As you settle in for a landing, you see a camel team maneuver a sled with a large anti-aircraft battery into position, where it is promptly shrouded in a tent that blends into the rest of the camp.`
```

			`	Traffic control directs you to a pad near a large door of the Shattered Light. Ruach and a small group of Remnant are gathered around a hologram of the ship, and seem to be co-ordinating teams exploring it. Large mounds of equipment appear to have expanded from the sides of the ship, some of which appears to be functional. As you go through your landing checks, the team thank you for the transportation and begin offloading their equipment.`
			`	"Welcome back, <first>!" signs Ruach from across the pad. "We have made considerable progress already." She appears set to explain what they have discovered.`
			choice
				`	"Great, but please spare me the details."`
					goto shatteredrequest
				`	"You figured out what happened? Great!"`
			`	She looks crestfallen "Well, we know what the ship thinks happened, anyway. It is likely that what actually happened differs significantly." Her posture takes on a more positive stance. "We have had the opportunity to start querying the onboard systems and piece together a little of the Shattered Light's history. Particularly the logs.`

> 			`	"As we all know, it was part of the exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`

Reviewer 6: ```suggestion
			`	"As we all know, it was part of the Exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`
```

			`	At this point another Remnant takes up the narrative. "Based on the initial incident logs, it appears that something in the stabilization fields failed during transit. The data is not clear whether the systems failed, or if something in the wormhole fluctuations exceeded the parameters that the systems could handle. In any case, the ship appears to have been shunted laterally from the wormhole in mid-transit.`

> 			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after, you spotted it."`

Reviewer 7: ```suggestion
			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with the prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after, you spotted it."`
```

			choice
				`	"Interesting. Well, let me know if I can help."`
					goto shatteredhelp
				`	"I am curious as to what you will do now."`
			`	"The first step will be to unpack the Shattered Light and set up all its equipment. Taking detailed scans of each piece as we do so, of course." She gestures around at the equipment expanding out of the side of the ship. "As you can see, we have already started the process and found that much of it is operational. One of the other Remnant continues. "The Shattered Light carried what amounts to an entire heavy industrial factory. Everything from smelters to welding equipment and everything in between." They smile and their gestures take on a tone of irony. "In fact, the ships we have now are in no small part due to the Shattered Light's absence. Without these capabilities, the choice about whether or not to commit to learning the techniques of those who lived here before us was much easier. If the Shattered Light had arrived on schedule we may well still be building ships out of the traditional materials prevalent at the time."`
				goto shatteredrequest
			label shatteredhelp

> 			`	"Thank you, captain. Of course we do not want to keep you waiting, so rest assured that we will contact you if your assistance is needed in the future."`

Reviewer 7: ```suggestion
			`	"Thank you, Captain. Of course we do not want to keep you waiting, so rest assured that we will contact you if your assistance is needed in the future."`
```

> 			`	"The first step will be to unpack the Shattered Light and set up all its equipment. Taking detailed scans of each piece as we do so, of course." She gestures around at the equipment expanding out of the side of the ship. "As you can see, we have already started the process and found that much of it is operational. One of the other Remnant continues. "The Shattered Light carried what amounts to an entire heavy industrial factory. Everything from smelters to welding equipment and everything in between." They smile and their gestures take on a tone of irony. "In fact, the ships we have now are in no small part due to the Shattered Light's absence. Without these capabilities, the choice about whether or not to commit to learning the techniques of those who lived here before us was much easier. If the Shattered Light had arrived on schedule we may well still be building ships out of the traditional materials prevalent at the time."`

Reviewer 6: ```suggestion
			`	"The first step will be to unpack the Shattered Light and set up all its equipment. Taking detailed scans of each piece as we do so, of course." She gestures around at the equipment expanding out of the side of the ship. "As you can see, we have already started the process and found that much of it is operational."`
			`	One of the other Remnant continues. "The Shattered Light carried what amounts to an entire heavy industrial factory. Everything from smelters to welding equipment and everything in between." They smile and their gestures take on a tone of irony. "In fact, the ships we have now are in no small part due to the Shattered Light's absence. Without these capabilities, the choice about whether or not to commit to learning the techniques of those who lived here before us was much easier. If the Shattered Light had arrived on schedule we may well still be building ships out of the traditional materials prevalent at the time."`
```

			label shatteredrequest

> 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`

Reviewer 7: ```suggestion
			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
```

> 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at Tycho crater... But if you have the resources and interest, the opportunity is there... The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`

Reviewer 7: ```suggestion
			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at Tycho Crater... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
```

> 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at Tycho Crater... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`

Reviewer 6: Tycho Crater isn't mentioned before this point in the story, so maybe shortly describe what it is.
```suggestion
			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
```

			`	"Sorry for leaving you behind, <first>, but this is just such an exciting discovery. This ship you found matches the profile of the Shattered Light - a large city ship that was part of the Exodus but never arrived on this side of the wormhole. There are so many questions to be answered! Anyway, the data is being distributed right now. Meet me in the cafeteria in a few hours to see what ideas have come up. Oh, and before I forget, we have allotted you <payment> for your assistance."`
			

mission "Remnant: Shattered Light 3"
	name "Ghost Touch"
	description "Help the Remnant experiment on the Shattered Light."
	blocked "As you approach the starport your commlink chimes with a message from Ruach requesting that you make space for 13 people as well as 19 tons of cargo."
	cargo "equipment" 19
	passengers 13
	source "Viminal"
	waypoint "Coluber"
	to offer
		has "Remnant: Shattered Light 2: done"

> 	npc "scan outfits"

Reviewer 6: This NPC block still needs moved below the on offer.

Author: Got it.

		government "Ember Waste"
		personality uninterested waiting staying mute
		system "Lucina"
		ship "Modified Dromedary Specter" "???"
		conversation
			action
				set "remnant: scanned specter"

> 			`It is on a crash course for Far Monad!" exclaims Ruach. "Quick, land there!`

Reviewer 6: This conversation could still use with a bit more meat.

Author: I believe I originally made it this short because dialog boxes look bad when they get more than a sentence or two in them; and conversation panels feel overbearingly huge and a break in flow when used in space. I think it might be in part because a dialog box is right there in the center, where the player's focus already is while in flight, since that's where their ship is; whereas the conversation box is over on the side. Sometimes *way* over to the side, depending on monitor resolution.

Reviewer 6: Yeah, I get why it was short as a dialog, but now as a conversation (since we need the `action` node) we should use some of the room we have.

Author: ![FKodLbg](https://user-images.githubusercontent.com/32169904/188284932-7e04127a-9f0a-4a96-bfdd-46b63a7ad615.png)

Author: I'm currently trying to write this expanded dialog.

For the future, though, could we get `action` stuff enabled for inside dialogs?

Reviewer 6: Probably not. The more appropriate change would be https://github.com/endless-sky/endless-sky/issues/5248,

Reviewer 4: I'm pretty sure there's an issue asking for it to be possible to perform game actions on NPC events, so it can just go outside the dialog/conversation.

Author: Well, we needed it here. Chances are good we'll need it again in the future, one way or another. It'd be nice if dialog boxes were more useful, anyway.

Author: New text added, so marking this resolved. Anything further here should probably be comments on that expanded text.

			`	Her gestures take on an appreciative tone. "The Maudrakoi made extensive use of the materials that make our ships and architecture so distinctive, and this has allowed their cities and technologies to survive the eons since they went extinct. One technology of note is they had an interesting ability to manipulate space. They used this to make large, highly complex objects, and then, for lack of a better term, partially embed them in pocket dimensions so as to not be entirely present. They used this frequently to allow equipment and materials to be much smaller than normal." She shrugs, and gestures with a frustrated tone. "We cannot replicate it, but we have figured out how to disrupt the compression, and thus recover the artifact."`
			`	The hologram shifts back to showing the Shattered Light. "This ship does not show the same signatures as the Maudrakoi artifacts, but we think it is sufficiently similar that our equipment should work on it too. Hopefully. In any case, though, time is passing, so we should start loading." With that, the Remnant gather their equipment and head out.`
				accept
	to complete
		never

mission "Remnant: Shattered Light 4"

> 	name "Shattered Light"
> 	description "Escort the wreck back to Viminal."
> 	landing

Reviewer 6: ```suggestion
	landing
	name "Shattered Light"
	description "Escort the wreck back to Viminal."
```

			`	"Before you leave, though, there was some stuff that survived intact we would like to share with you." Behind her five androids emerge from one of the cargo holds, each one carrying a large trunk of equipment.`
			scene "outfit/android"
			`	 "These are known as NDR-114 Androids, and were cutting edge innovations back when the Exodus occurred. Don't expect too much of them. They are not self-aware, sentient, or even particularly advanced artificial intelligences. They are, however, excellent extensions of a ship's self-maintenance systems. As it turns out, a large number of these androids were instrumental in keeping the Shattered Light flying all these years. We have plenty to study, so we thought you might appreciate having a few." She makes a dismissive gesture. "If they end up not being helpful for you, our outfitters will be able to re-assign them to another ship.`
			`	"Embers light your path, Captain <first>." With that, the crowd disperses.`

mission "Remnant: Shattered Light 5"

> 	name "Brilliant Energy"
> 	description "Retrieve a team with their equipment from <destination> to help research the Shattered Light."
> 	landing

Reviewer 6: ```suggestion
	landing
	name "Brilliant Energy"
	description "Retrieve a team with their equipment from <destination> to help research the Shattered Light."
```

	source "Viminal"
	destination "Caelian"
	to offer
		has "Remnant: Shattered Light 4: done"

mission "Remnant: Shattered Light 6"

> 	name "Shattered Light"
> 	description "Ferry the team and their gear back to <destination>"
> 	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 tons of cargo."
> 	cargo "equipment" 18
> 	passengers 11
> 	landing

Reviewer 6: ```suggestion
	landing
	name "Shattered Light"
	description "Ferry the team and their gear back to <destination>"
	blocked "From your count at the information desk, you estimate that you need space for 11 people as well as 18 tons of cargo."
	cargo "equipment" 18
	passengers 11
```

		conversation
			action
				set "remnant: scanned specter"
			`As the scan results arrive the data is relayed down to the team in the cargo hold. Your video feed shows a large hologram of the system floating in midair above one of the pallets. Nearby, terminals flash rolling readouts and analyses. As the data on the spectral ship solidifies, cones of projected courses appear on the display. Line by line the cone tightens in as additional data flows from your telemetry, closing in on a path intersecting with the nearby world.`
			`	While the system continues to refine the orbital paths, the team seemed disinclined to wait: Ruach activates her commlink, broadcasting quickly to you. "It is on a crash course for Far Monad!" she exclaims. "Quick, land there!" Behind her, you can see her team scrambling to move equipment around near your cargo door and seal their spacesuits.`
	to complete
		never

mission "Remnant: Shattered Light 4"
	landing
	name "Shattered Light"
	description "Escort the wreck back to Viminal."
	source "Far Monad"
	destination "Viminal"
	to offer
		has "remnant: scanned specter"
	on offer
		fail "Remnant: Shattered Light 3"
		conversation

> 			`With the Shattered Light careening towards the surface of Far Monad, you push the <ship> into a dive. As you focus on finding a safe place to land within close proximity of the approaching ship, you notice out the corner of your eye that the Remnant team have all suited up and are pushing their equipment to the cargo hatch. A message appears on your console.`

Reviewer 5: Seems a bit... cliche? Action movie-y? Not sure if this is silly or cool - probably a bit of both. Might make a decent episode of Star Trek.

Does Primary Weft gravity also affect objects in the Secondary Weft? What are the interactions between objects on the two wefts? Are the two merely special 3d planes in a 4d universe?

			choice
				`	"I have things to do, so just let me know if I can help."`
					goto shatteredhelp
				`	"So, what happened?"`
			`	The team of Remnant are almost too excited for you to understand their gestures, but one calms down enough that their signing is comprehensible. "To be honest, we were expecting this to be the first of several trips doing detailed scans and trying out different sorts of synchronization fields. But when we saw that it was heading to the planet..." She trails off, and another one continues. "We already had the sensors running as we approached, so we jumped and triggered the fields as soon as we got a reasonable match."`
			`	He frowns a bit. "It worked, although not perfectly. There are chunks missing, some of which seriously impair its functionality. But it is back in the Primary Weft, so now we have time to work on it and figure out what happened to it.`
			choice
				`	"I think I get the picture. Let me know if I can help."`
					goto shatteredhelp
				`	"What is this 'Primary Weft' you have mentioned?"`

> 			`	Another Remnant steps forward. "The Weft is what we term the mesh of dimensions, energy, and time that form a subset of the universe. A layer, of sorts." He looks thoughtful. "What you probably know as 'Hyperspace' is what we technically call the Secondary Weft. There are a few others we are fairly confident exist, and several that we have hypothesized might exist but we have no means of confirming. Practically, each of these overlaps with the Primary Weft to some extent." His gestures take on an amused tone. "Of course, what we call the Primary Weft is only primary to those of us who live here. For all we know, there may be people living elsewhere that consider their own weft to be the primary one. Nothing says ours is special or the best or anything of the sort. Actually, it has been proposed that we call the Primary Weft the Warp. It is an old weaving term that serves as a reasonable analogy to how the warp and weft interact, and might better reflect its status instead of being labeled 'Primary.'`

Reviewer 5: Been reading Banks lately? I like it, it fits well with how much of the other game lore is Culture themed.

The introduction of the wefts opens up a lot of questions about ES's cosmology. I would find it valuable if First Last, and by extent the player, had the option to ask a cosmologist about their theories of the universe. Maybe Ijs knows someone, or maybe First Last can browse the data chip from Alexandria for relevant texts.

					goto shatteredrequest
				`	"You figured out what happened? Great!"`
			`	She looks crestfallen "Well, we know what the ship thinks happened, anyway. It is likely that what actually happened differs significantly." Her posture takes on a more positive stance. "We have had the opportunity to start querying the onboard systems and piece together a little of the Shattered Light's history. Particularly the logs.`
			`	"As we all know, it was part of the Exodus fleet that entered the Terminus wormhole. Unlike most of the other ships, however, it never emerged from the other side. Fortunately, it had a significantly larger array of instruments than was normal for ships of its type, and it looks like at least some of the databanks survived."`
			`	At this point another Remnant takes up the narrative. "Based on the initial incident logs, it appears that something in the stabilization fields failed during transit. The data is not clear whether the systems failed, or if something in the wormhole fluctuations exceeded the parameters that the systems could handle. In any case, the ship appears to have been shunted laterally from the wormhole in mid-transit.`

> 			`	"It is unclear what happened after that, but the ship was subject to significant forces including unidentified impacts that caused multiple system failures. The data loses some coherence at this point, but it basically seems to have been semi-permanently stuck out-of-phase with the prime weft until an energy surge destabilized it and it wobbled back into prime coherence. At that point, it started being visible again, and shortly after, you spotted it."`

Reviewer 5: How long did this take? We can infer from the SL's heavily modified state that they were out of phase for a while, but more details on the ship's time in the secondary weft would be good.

Author: Lots of questions. :) 

At this point, this is just the preliminary report. A "here's what we got out of our initial pass" sort of thing. There will be more story in the future, including opportunities to ask more questions.
